### DISCUSSION TITLE
Textadept Zettelkasten module

### DISCUSSION THREAD

**Discussion by sthesing** [orbitalquark/textadept#186] [Show and tell]
I have been using Textadept as my primary tool working with Niklas Luhmann's Zettelkasten-method for a few years now. In fact, the whole idea to write my own Zettelkasten software came from my wish to edit my notes in my favourite text editor: Textadept.
Initially, the tool was a patchy python script and a bit of even patchier lua code to accompany it.

Long story short, I rewrote my zettelkasten-tool a while back and now I finally found the time to release the accompanying textadept module. Check it out, if you're interested:

- The module: [ta-zettels-ng](https://gitlab.com/sthesing/ta-zettels-ng)
- The command line tool used by the module: [Zettels](https://gitlab.com/sthesing/zettels)

And thanks for an amazing editor :-)

**Comment by orbitalquark** [comment]
Cool! Thanks for sharing :)
